### DISCUSSION TITLE
Random Counter Increments

### DISCUSSION THREAD

**Discussion by martin072** [arendst/Tasmota#22215] [Q&A]
I have a question. I am using this "DC 5V NPN NO LJ18A3-8-Z/BX-5V"  sensor for a water meter. I've configured the debouncing times and also configured it as a COUNTER (with pull-up I assumed, there is no hardware pull-up, and the 5v input is taken care of with a voltage divider), but on random occasions (not every minute, more in hours / days) it triggers the Counter? Could that be the sensor it self, or do I need to configure / change something in Tasmota?

**Comment by lalo-uy** [comment]
Random interferece could happen.
Don’t know the frequency of you pulses, but a 100nf between the input and
groun always helps.

**Reply by martin072** [reply]
Thanks. Will try. It’s for counting the pulses on a water meter so most of the time it does nothing and is idle.

**Reply by sfromis** [reply]
You could also try if tweaking the CounterDebounce settings help.

**Reply by martin072** [reply]
Sorry for the late reply, @sfromis What would be the ideal settings to try? one rotation would likely not take more than 2-3sec. This is what's configured now:

```
"CounterDebounce":500
"CounterDebounceLow":50
"CounterDebounceHigh":50
```

**Reply by sfromis** [reply]
There is no "ideal", all depends on your hardware and what happens on the wires.
